tests/stress/stress_test.go

```diff
@@ -77,143 +71,6 @@ func TestMain(m *testing.M) {
-	require.NoError(t, err)
-
-	block := new(modules.ChainBlockResponse)
-	err = utils.DecodeRPC(t, respBody, block)
-	if err != nil {
-		return nil
-	}
-
-	header := block.Block.Header
-
-	parentHash, err := common.HexToHash(header.ParentHash)
-	require.NoError(t, err)
-
-	nb, err := common.HexToBytes(header.Number)
-	require.NoError(t, err)
-	number := big.NewInt(0).SetBytes(nb)
-
-	stateRoot, err := common.HexToHash(header.StateRoot)
-	require.NoError(t, err)
-
-	extrinsicsRoot, err := common.HexToHash(header.ExtrinsicsRoot)
-	require.NoError(t, err)
-
-	digest := [][]byte{}
-
-	for _, l := range header.Digest.Logs {
-		var d []byte
-		d, err = common.HexToBytes(l)
-		require.NoError(t, err)
-		digest = append(digest, d)
-	}
-
-	h, err := types.NewHeader(parentHash, number, stateRoot, extrinsicsRoot, digest)
-	require.NoError(t, err)
-
-	b, err := types.NewBodyFromExtrinsicStrings(block.Block.Body)
-	require.NoError(t, err, fmt.Sprintf("%v", block.Block.Body))
-
-	return &types.Block{
-		Header: h,
-		Body:   b,
-	}
-}
-
-// headerResponseToHeader converts a *ChainBlockHeaderResponse to a *types.Header
-func headerResponseToHeader(t *testing.T, header *modules.ChainBlockHeaderResponse) *types.Header {
-	parentHash, err := common.HexToHash(header.ParentHash)
-	require.NoError(t, err)
-
-	nb, err := common.HexToBytes(header.Number)
-	require.NoError(t, err)
-	number := big.NewInt(0).SetBytes(nb)
-
-	stateRoot, err := common.HexToHash(header.StateRoot)
-	require.NoError(t, err)
-
-	extrinsicsRoot, err := common.HexToHash(header.ExtrinsicsRoot)
-	require.NoError(t, err)
-
-	digest := [][]byte{}
-
-	for _, l := range header.Digest.Logs {
-		var d []byte
-		d, err = common.HexToBytes(l)
-		require.NoError(t, err)
-		digest = append(digest, d)
-	}
-
-	h, err := types.NewHeader(parentHash, number, stateRoot, extrinsicsRoot, digest)
-	require.NoError(t, err)
-	return h
-}
-
-// getHeader calls the endpoint chain_getHeader
-func getHeader(t *testing.T, node *utils.Node, hash common.Hash) *types.Header { //nolint
-	respBody, err := utils.PostRPC(t, chain_getHeader, endpoint(node), "[\""+hash.String()+"\"]")
-	require.NoError(t, err)
-
-	header := new(modules.ChainBlockHeaderResponse)
-	utils.DecodeRPC(t, respBody, header)
-	return headerResponseToHeader(t, header)
-}
-
-// getChainHead calls the endpoint chain_getHeader to get the latest chain head
-func getChainHead(t *testing.T, node *utils.Node) *types.Header {
-	respBody, err := utils.PostRPC(t, chain_getHeader, endpoint(node), "[]")
-	require.NoError(t, err)
-
-	header := new(modules.ChainBlockHeaderResponse)
-	utils.DecodeRPC(t, respBody, header)
-	return headerResponseToHeader(t, header)
-}
-
-// compareChainHeads calls getChainHead for each node in the array
-// it returns a map of chainHead hashes to node key names, and an error if the hashes don't all match
-func compareChainHeads(t *testing.T, nodes []*utils.Node) (map[common.Hash][]string, error) {
```

Comment:
is there tests/rpc package using this? if not, I think we can leave it here

Reply:
If you are planning to reuse it, moving to utils makes sense, either works for me for now

Reply:
will move it back then, not using it anywhere for now...